msticpy/context/vtlookupv3/vtlookupv3.py

```diff
@@ -687,7 +678,7 @@ def lookup_iocs_relationships(
         relationship: str,
         observable_column: str = ColumnNames.TARGET.value,
         observable_type_column: str = ColumnNames.TARGET_TYPE.value,
-        limit: Optional[int] = None,
+        limit: int = None,
```

Comment:
Not clear why you're changing the type annotation here - if it's a nullable int it should be Optional[int]

Reply:
Acknowledged - Please see changes [in commit:817c6607afd96b6dee427e10c9c36bc733a13b77](https://github.com/microsoft/msticpy/pull/739/commits/817c6607afd96b6dee427e10c9c36bc733a13b77)